lib/view/component/sentence/linked_annotation.dart

```diff
@@ -0,0 +1,50 @@
+import 'package:flutter/material.dart';
+
+import 'package:url_launcher/url_launcher.dart';
+
+import 'package:hakondate/constant/app_color.dart';
+import 'package:hakondate/constant/size.dart';
+
+class LinkedAnnotation extends StatelessWidget {
+  const LinkedAnnotation ({
+    required this.text, 
+    required this.url,
+    super.key,
+    });
+
+    final String text;
+    final String url;
+
+  @override
+  Widget build(BuildContext context) {
+    return InkWell(
+      onTap: () async {
+        if (await canLaunchUrl(
+          Uri.parse(url),
+        )) {
+          await launchUrl(
+            Uri.parse(url),
+              mode: LaunchMode.inAppWebView,
+          );
+        }
+      },
+      child: Column(
+        // ignore: always_specify_types
+        children: [
```

Comment:
`// ignore: ~~~~` はlinterを無視するという文句なので避ける
```suggestion
        children: <Widget>[
```